.gitignore

```diff
@@ -45,3 +45,6 @@ output/*/index.html
 
 # Sphinx
 docs/_build
+
+# Sqlite databases
+*.sqlite3
```

Comment:
this is probably also a left-over from sqlite testing?

Reply:
✔️